sound/soc/sof/topology.c

```diff
@@ -1596,6 +1597,17 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 
 	swidget->private = volume;
 
+	list_for_each_entry(scontrol, &sdev->kcontrol_list, list) {
+		if (scontrol->comp_id == swidget->comp_id) {
+			volume->min_value =
+			scontrol->volume_table[scontrol->min_volume_step];
+			volume->max_value =
+			scontrol->volume_table[scontrol->max_volume_step];
+			volume->channels = scontrol->num_channels;
```

Comment:
the indentation is just horrible. try to do something here and worst case break the 80 char rule.

Reply:
@plbossart added an intermediate variable to solve this.